### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by louisezetterlund**
Added more information to the README. #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # CI Server
 
+**This is a Continuous Integration server that automatically runt tests on code that are pushed to GitHub.**
```
Comment (EmilGedda) [inline README.md]: [...] that can automatically compile and run tests on code [...]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+
+The server runs jUnit tests triggered by a push-event on GitHub. The code is tested and the program generates documentation for the classes.
```
Comment (EmilGedda) [inline README.md]: Första meningen stämmer inte, ta bort.
Andra meningen: "The server is unit tested using jUnit and uses javadoc for documentation. The documentation is automatically updated and deployed to GitHub Pages on every commit or merge to master."

File: README.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+The test results are sent back to GitHub where the commit status gets updated to either Success or Failure.
```
Comment (EmilGedda) [inline README.md]: "The test results [...]" stämmer inte. "The commit status of a commit gets updated to Success or Failure depending on the exit code on the bash process". Flytta sen den meningen ner till sist efter nästa stycke.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+
+When our server is notified of a push-event, we create a temporary folder to which we save the path. We clone the repository in question into that folder and look for an instruction file in the form of a bash file.
```
Comment (EmilGedda) [inline README.md]: push event. Använd inte we och our, kör "When the server [...]" etc. Bisatsen i första meningen kan tas bort, och "that folder" can bytas ut mot "a temporary folder". "and look for [...]" borde skrivas som "and then tries to execute a bash file called `executes.sh` which should contain the steps for compiling and testing the project.".

File: README.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+We then run the bash instructions on the file. The bash file contains all the instructions
+necessary to run the tests. After we've run the file, we save the exit code to see if the tests failed or succeeded.
```
Comment (EmilGedda) [inline README.md:12]: Dom här kan tas bort och bytas ut mot meningen som som skulle flyttas ned

**Review by arohlen — APPROVED** [review]
Very good, you solved all the problems!!

**Pull request merged**